SimPEG/optimization.py

```diff
@@ -581,11 +596,18 @@ def modifySearchDirection(self, p):
             :return: (xt, passLS) numpy.ndarray, bool
         """
         # Projected Armijo linesearch
-        self._LS_t = 1
+        self._LS_t = 1.
         self.iterLS = 0
         while self.iterLS < self.maxIterLS:
             self._LS_xt = self.projection(self.xc + self._LS_t * p)
-            self._LS_ft = self.evalFunction(self._LS_xt, return_g=False, return_H=False)
+            self._LS_ft, self._LS_ft_descent = self.evalFunction(
```

Comment:
in order to make this optional, would it be better instead to loop through the `self.stoppersLS` and have this be a method on each stopper?